.infrastructure/security/rbac.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pods-lister
+  namespace: todoapp
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pods-lister
+  namespace: todoapp
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["list"]

```

Comment:
According to the task description, this role should have access to secret 
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_12_rbac/assets/87720583/70a6bd08-fd85-4354-9bde-37237bcf3742)